Editor/ObjectMapping/ObjectMappingContext.cs

```diff
@@ -167,6 +177,17 @@ public override bool TryMapProperty(string property, out API.MappedPropertyInfo
                 return true;
 
             }
+            
+            public override bool TryGetMappedVrmFirstPersonFlag(out VrmFirstPersonFlag vrmFirstPersonFlag)
+            {
+                if (!_info.VrmFirstPersonFlag.HasValue)
+                {
+                    vrmFirstPersonFlag = default;
+                    return false;
+                }
+                vrmFirstPersonFlag = _info.VrmFirstPersonFlag.Value;
```

Comment:
```suggestion
                if (_info.VrmFirstPersonFlag is not { } firstPersonFlag)
                {
                    vrmFirstPersonFlag = default;
                    return false;
                }
                vrmFirstPersonFlag = firstPersonFlag;
```

同様にHasValue / Valueは避けたいです

Reply:
switch の方が読みやすく感じたのでそうしました。 https://github.com/anatawa12/AvatarOptimizer/pull/1376/commits/d3d21e41f2ce74d9950280d26ffef062bd9be5e9